stremio-core-web/src/model/serialize_ctx.rs

```diff
@@ -30,6 +30,15 @@ mod model {
         pub notifications: Notifications<'a>,
         pub search_history: Vec<SearchHistoryItem<'a>>,
         pub events: &'a Events,
+        pub streaming_server_urls: Vec<StreamingServerUrlItem>,
+    }
+
+    #[derive(Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct StreamingServerUrlItem {
+        pub url: String,
+        #[serde(rename = "_mtime")]
```

Comment:
- [ ] this renaming is not necessary. For libraryItem it;'s renamed for historical reasons. This is not the case for this filed